src/Viewer.js

```diff
@@ -307,12 +307,12 @@ export class Viewer {
                 this.rootElement.style.position = 'absolute';
                 document.body.appendChild(this.rootElement);
             } else {
-                this.rootElement = this.renderer.domElement || document.body;
+                this.rootElement = this.renderer.domElement.parentElement || document.body;
```

Comment:
Not sure they could always be a parent element. In any case this seems unrelated to the changeset.